azure-pipelines.yml

```diff
@@ -247,6 +228,30 @@ jobs:
     condition: eq(variables['Build.Reason'], 'PullRequest')
     displayName: Check Licence Headers
 
+- job: slow_lints
+  displayName: Perform Slower Lints
+  dependsOn: lint
+  pool:
+    vmImage: ubuntu-18.04
+  steps:
+  - template: ci/install-package-dependencies.yml
+  - bash: |
+      # Here we look for all *.vendor.hjson files in the repo to re-vendor them.
+      # We exclude the following:
+      # - Any in 'hw/vendor/lowrisc_ibex', because that directory is vendored.
+      find . \
+        -not \( -path './hw/vendor/lowrisc_ibex' -prune \) \
+        -name '*.vendor.hjson' \
+        | xargs -n1 util/vendor.py --verbose \
+        && git diff --exit-code
+      if [[ $? != 0 ]]; then
+        echo -n "##vso[task.logissue type=error]"
+        echo "Vendored repositories not up-to-date. Run util/vendor.py to fix."
+        exit 1
+      fi
+    condition: always()
```

Comment:
You can remove this line for the new job (or leave it in there for forward-compat).

Reply:
I thought I'd leave it in for forward-compat - we want this to run even if other lints fail, is my understanding.